### DISCUSSION TITLE
Pre-Trained Models in DPA2 as a Potential Alternative to Active Learning with DP-GEN?

### DISCUSSION THREAD

**Discussion by poyaii** [deepmodeling/deepmd-kit#4093] [Q&A] [answered]
Hello. I have an important question that's been on my mind. Can using pre-trained models in DPA2 be a good alternative to active learning and using DP-GEN? Thank you.

**Comment by njzjz** [comment]
I don't think pre-training and active learning are in conflict. They can be used at the same time.

**Reply by poyaii** [reply]
Thank you for your response. If I am unable to use DP-GEN due to challenges in configuring it on my university's cluster, can I use the DPA model instead to improve the accuracy and performance of my model?

**Reply by njzjz** [reply, marked as answer]
I believe no one has shown that the pre-training can replace the active learning, meaning you cannot assume it (whether it is true or false) when writing papers.
